src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -259,7 +259,7 @@ export default function MissingFileOverview({
   // Update sponsor to identify missing uploads
   const sponsorConditionalPages = getConditionalPages(pages, data);
   const sponsorMiss = {
-    name: data?.[nonListNameKey || 'veteransFullName'],
+    [nonListNameKey ?? 'name']: data?.[nonListNameKey || 'veteransFullName'],
```

Comment:
Fix for bug that prevented applicant names from showing in missing file overview page - the issue was that the key was always set to `name`, which wasn't correct when using the optional `nonListNameKey` property:

|Before|After|
|-|-|
|![Screenshot 2024-12-20 at 16 07 28](https://github.com/user-attachments/assets/8d55fa9a-ae86-4451-b455-a9a02d55cf3d)|![Screenshot 2024-12-20 at 16 08 09](https://github.com/user-attachments/assets/ca749c07-1a61-4856-80da-b2c0f79b29f6)|